packages/backend/src/controllers/organisme/update.ts

```diff
@@ -1,11 +1,19 @@
+import type { NextFunction, Response } from "express";
+
+import { UserRequest } from "../../types/request";
+
 const Organisme = require("../../services/Organisme");
 const AppError = require("../../utils/error");
 
 const logger = require("../../utils/logger");
 
 const log = logger(module.filename);
 
-module.exports = async function update(req, res, next) {
+module.exports = async function update(
```

Comment:
```suggestion
export default async function update(
```
attention aussi à ne pas laisser de `require()` tout doit passer en `import`